### PR TITLE
Make fab staging, production, and preview use inventory file

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
love this change. :+1:

**Comment by dannyroberts** [comment]
@millerdev FYI. I'm going to test this on staging, but since I had to do something manually for each target, that doesn't fully ensure nothing goes wrong with production deploy. If something does go wrong it's most likely to just fail. I can run a production preindex to minimize the chances of an unexpected error being dumped in your lap, but I just wanted you to be aware of this in case you do have to debug something.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dannyroberts** [comment]
@dimagi/team-commcare-hq Note that this means you'll have to install ansible from [dev-requirements.txt](https://github.com/dimagi/commcare-hq/commit/8429b7a6eb0c11b122a7ed113f1c248ff55488a5) in order to deploy from now on

File: fab/fabfile.py [hunk 1]
```diff
@@ -339,44 +280,12 @@ def staging():
...
+    execute(development)
 
     env.django_command_prefix = '%(virtualenv_root)s/bin/newrelic-admin run-program ' % env
     env.supervisor_env_vars = 'NEW_RELIC_CONFIG_FILE=../newrelic.ini,NEW_RELIC_ENVIRONMENT=staging'
 
-@task
-def realstaging():
-    print "(You know you can just use 'staging' now, right? Doing that for ya.)"
-    staging()
-
```
Comment (esoergel) [inline fab/fabfile.py]: haha yeah, probably no longer necessary

**Comment by esoergel** [comment]
Should the inventory files have a `.yml` extension on them or something?

**Comment by dannyroberts** [comment]
good question; i'm trying to follow ansible's convention, and they don't mark them so whether it's right or not... at least it was intentional!

**Comment by dannyroberts** [comment]
Thanks!

**Comment by millerdev** [comment]
Beautiful!

**Pull request merged**